sender/sms/sms.go

```diff
@@ -1,11 +1,27 @@
 package sms
 
 import (
+	"errors"
+	"fmt"
 	gotell "github.com/ftpsolutions/go-tell"
 	"github.com/ftpsolutions/go-tell/sender"
+	"regexp"
+	"strings"
 )
 
 func validateSMS(job *gotell.Job) error {
+	re, _ := regexp.Compile(`^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$`)
```

Comment:
It would be better to use `MustCompile` here;
https://golang.org/pkg/regexp/#MustCompile

Something like this

```suggestion
var re = regexp.MustCompile(`^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$`)

func validateSMS(job *gotell.Job) error {
```

Reply:
Ooh, TIL- nice